psamm/commands/robustness.py

```diff
@@ -78,20 +78,20 @@ def run(self):
         else:
             reaction = self._model.get_biomass_reaction()
             if reaction is None:
-                raise CommandError('The biomass reaction was not specified')
+                self.fail('The biomass reaction was not specified')
```

Comment:
There are two places that I think should keep the way it is. See the comments I made above. If you agree, I would git push a new version.